src/pk-transaction.c

```diff
@@ -3059,7 +3059,7 @@ pk_transaction_get_details_local (PkTransaction *transaction,
 			g_set_error (&error,
 				     PK_TRANSACTION_ERROR,
 				     PK_TRANSACTION_ERROR_NO_SUCH_FILE,
-				     "No such file %s", full_paths[i]);
+				     "No such file %s, or unknown or unsupported content type", full_paths[i]);
```

Comment:
It's a bit wordy, perhaps `% is not found or unsupported`

Reply:
I added `File %s is not found or unsupported` in front of that so that we don't start the sentence with a path, and add context to the message that this was a file path.